composer.json

```diff
@@ -16,6 +16,7 @@
         "equip/structure": "^1.0",
         "filp/whoops": "^2.0",
         "lukasoppermann/http-status": "^2",
+        "monolog/monolog": "^1.19",
```

Comment:
We shouldn't require monolog specifically, but rather [`psr/log`](https://packagist.org/packages/psr/log).